src/FriendlyErrors.php

```diff
@@ -48,10 +48,15 @@ public static function checkTesseractPresence($executable)
 
 	public static function checkCommandExecution($command, $stdout, $stderr)
 	{
-		$file = $command->getOutputFile();
-
-		if (($command->useFileAsOutput && file_exists($file) && filesize($file) > 0) ||
-			(!$command->useFileAsOutput && $stdout)) return;
+		if ($command->useFileAsOutput) {
+			$file = $command->getOutputFile();
+			$executionCheckOk = (file_exists($file) && filesize($file) > 0);
+		}
+		else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method checkCommandExecution uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://app.codacy.com/manual/thiagoalessio/tesseract-ocr-for-php/pullRequest?prid=4826731)